services/validatorWorker/follower.js

```diff
@@ -97,7 +128,7 @@ function getLatestMsg(channelId, from, type) {
 		from: from,
 		'msg.type': type,
 	})
-	.sort({ _id: -1 })
+	.sort({ 'msg.created': -1 })
```

Comment:
great! I think there one "@TODO" comment that can be deleted because this is fixed now

Reply:
its deleted